operators/multiclusterobservability/controllers/placementrule/placementrule_controller.go

```diff
@@ -116,6 +118,15 @@ func (r *PlacementRuleReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{}, nil
 	}
 
+	// Do not reconcile objects if this instance of mch has the
+	// `disableAddonAutomaticInstallationAnnotationKey` annotation
+	if value, ok := mco.GetAnnotations()[disableAddonAutomaticInstallationAnnotationKey]; ok &&
```

Comment:
The annotation should be checked on each of the `ManagedCluster` objects, not on MCO custom resource.

Reply:
Makes sense. I'll update that and move it into the predicate per slack discussion